modules/router/templates/assets_origin.conf.erb

```diff
@@ -6,8 +6,11 @@ server {
 }
 
 server {
+  <%- if scope.lookupvar('::aws_migration') %>
+  server_name <%= @vhost_name -%> <%= @vhost_name -%>.* <%= @vhost_aliases.join(' ') -%>;
+  <%- else %>
   server_name <%= @vhost_name -%> <%= @vhost_aliases.join(' ') -%>;
-
+  <%- end %>
```

Comment:
You'll be removing a white line in the current configs

Reply:
Hmm, these conditionals are a bit strange anyway. I think they may do stuff I don't expect.

Reply:
I've updated this so the conditional is in the right place